### PR TITLE
Resolved #8002, added new constant.

### LINKED ISSUE #8002: Provide Constants for Max Files and Max Embeds When Sending a Message
### Summary

Provide constants for the max number of files and of embeds a message can be sent with.

### What is the feature request for?

The core library

### The Problem

When sending a message, the number of embeds and files are limited to 10, and an exception is thrown if you exceed that. So we have to make sure we don't go over the limit.

### The Ideal Solution

I would like there to be a constant I can refer to that has the limit so I don't have to hardcode it myself and remember to update it if it ever changes.

### The Current Solution

The current solution is to hardcode in the limits and hope they don't change (which, to be fair, I don't think they ever have).

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by farkon00**
## Summary

Resolves issue #8002, asking for new constant.

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/http.py [hunk 1]
```diff
@@ -51,11 +51,13 @@
 import aiohttp
 
 from .errors import HTTPException, Forbidden, NotFound, LoginFailure, DiscordServerError, GatewayNotFound
+from .embeds import MAX_MESSAGE_EMBEDS
 from .gateway import DiscordClientWebSocketResponse
 from .file import File
 from .mentions import AllowedMentions
 from . import __version__, utils
 from .utils import MISSING
+print(__name__)
```
Comment (DJJ05) [inline discord/http.py:60]: Did you forget to remove this?

Comment (farkon00) [reply discord/http.py:60]: @DJJ05 Sorry, yes, I forgot, I will fix it, one moment.

Comment (farkon00) [reply discord/http.py]: Fixed

**Review by DJJ05 — APPROVED** [review]

File: discord/http.py (hunk 1 above)
Comment (DJJ05) [reply discord/http.py]: 👍

**Comment by GoogleGenius** [comment]
Might want to fix the title by the way :+1:

**Comment by farkon00** [comment]
@GoogleGenius Its just the same thing.

> the number of embeds and files are limited to 10

**Comment by GoogleGenius** [comment]
> @GoogleGenius Its just the same thing.
> 
> > the number of embeds and files are limited to 10

Ah alright thanks for letting me knw!

**Comment by Rapptz** [comment]
Heya,

Thanks for the PR. Unfortunately I'm not really sure if I want to expose a constant for this. I'm also not really sure I want to expose it in this manner too. I figure hypothetical constants would belong in their own separate submodule with proper documentation. However I'm still not sure on how useful such as hypothetical submodule would be.

**Pull request closed without merging**